### PR TITLE
fix few comments, free local refs

### PR DESCRIPTION
**Pull request by Suvitruf**
1) Some typo fix.
2) Clear local refs. https://developer.android.com/training/articles/perf-jni#local-and-global-references
3) float comparison fix

### CODE REVIEW

**Comment by Suvitruf** [comment]
I've signed agreement.

**Comment by Suvitruf** [comment]
Comparing floats in general with precision is a good practice.

> There is no calculations for those values

But, `AvgSessonLength` is public. So, someone could change it, no? Also, by default it's `0f`. If it's never changed, the method `HasAvgSessonLength` doesn't make any sense, it always return `true`.

**Comment by olehkuznetsov** [comment]
> But, `AvgSessonLength` is public. So, someone could change it, no?
This object is returned from server and it's assumed as used for just read information from it.
Some values can be returned as 'UNSET_VALUE' and it means their values are unknown on server. There is no any calculation on the server which could have a result -1(UNSET_VALUE) only if it is set explicitly.

**Comment by Suvitruf** [comment]
Do you want me to revert this commit? I mean, this changes in PlayerStats.cs doesn't break anything, right? So,,,

File: source/PluginDev/Assets/GooglePlayGames/Platforms/Android/Developers/JavaObjWrapper.cs [hunk 1]
```diff
@@ -291,52 +319,59 @@ public T InvokeCall<T>(string name, string sig, params object[] args)
             IntPtr method = AndroidJNI.GetMethodID(RawClass, name, sig);
             jvalue[] jArgs = ConstructArgArray(args);
 
-            if (method == IntPtr.Zero)
+            try 
             {
-                Debug.LogError("Cannot get method for " + name);
-                throw new Exception("Cannot get method for " + name);
-            }
+                if (method == IntPtr.Zero)
```
Comment (olehkuznetsov) [inline source/PluginDev/Assets/GooglePlayGames/Platforms/Android/Developers/JavaObjWrapper.cs]: Could we have this check before line:
jvalue[] jArgs = ConstructArgArray(args);

File: source/PluginDev/Assets/GooglePlayGames/Platforms/Android/Developers/JavaObjWrapper.cs [hunk 2]
```diff
@@ -176,20 +183,27 @@ protected static jvalue[] ConstructArgArray(object[] theArgs)
             jvalue[] jArgs = ConstructArgArray(args);
             IntPtr val = AndroidJNI.CallStaticObjectMethod(rawClass, method, jArgs);
```
Comment (olehkuznetsov) [inline source/PluginDev/Assets/GooglePlayGames/Platforms/Android/Developers/JavaObjWrapper.cs]: > IntPtr val = AndroidJNI.CallStaticObjectMethod(rawClass, method, jArgs);

Should this line be in try block too?

File: source/PluginDev/Assets/GooglePlayGames/BasicApi/Events/IEvent.cs [hunk 3]
```diff
@@ -16,7 +16,7 @@
 
 namespace GooglePlayGames.BasicApi.Events
 {
-
+    using GooglePlayGames.Native.PInvoke;
```
Comment (olehkuznetsov) [inline source/PluginDev/Assets/GooglePlayGames/BasicApi/Events/IEvent.cs]: >  using GooglePlayGames.Native.PInvoke

Can't see usage of this in this file.

**Comment by olehkuznetsov** [comment]
> Do you want me to revert this commit? I mean, this changes in PlayerStats.cs doesn't break anything, right? So,,,

It would be great if we revert it from this pull request as I really like other changes in this one ))

File: source/PluginDev/Assets/GooglePlayGames/BasicApi/Events/IEvent.cs (hunk 3 above)
Comment (Suvitruf) [reply source/PluginDev/Assets/GooglePlayGames/BasicApi/Events/IEvent.cs]: `EventManager` is from this namespace. If we remove this `using`, we have to specify full name for `EventManager` in `cref="EventManager"` annotation of `IEvent`.

File: source/PluginDev/Assets/GooglePlayGames/Platforms/Android/Developers/JavaObjWrapper.cs (hunk 2 above)
Comment (Suvitruf) [reply source/PluginDev/Assets/GooglePlayGames/Platforms/Android/Developers/JavaObjWrapper.cs]: Probably, yes)

File: source/PluginDev/Assets/GooglePlayGames/Platforms/Android/Developers/JavaObjWrapper.cs (hunk 1 above)
Comment (Suvitruf) [reply source/PluginDev/Assets/GooglePlayGames/Platforms/Android/Developers/JavaObjWrapper.cs]: Yeh, nice point.

**Comment by Suvitruf** [comment]
I've pushed this minor changes and reverted changes in PlayerStats.cs

File: source/PluginDev/Assets/GooglePlayGames/BasicApi/Events/IEvent.cs (hunk 3 above)
Comment (Suvitruf) [reply source/PluginDev/Assets/GooglePlayGames/BasicApi/Events/IEvent.cs]: I've removed `using` and specified full name in annotation.

**Pull request merged**